src/Components/v2/CollectionsHubsNav.tsx

```diff
@@ -1,19 +1,32 @@
-import { CSSGrid } from "@artsy/palette"
-import { Serif } from "@artsy/palette"
+import { CSSGrid, Serif } from "@artsy/palette"
 import { CollectionsHubsNav_marketingHubCollections } from "__generated__/CollectionsHubsNav_marketingHubCollections.graphql"
-import { AnalyticsSchema } from "Artsy/Analytics"
+import * as Schema from "Artsy/Analytics/Schema"
 import { useTracking } from "Artsy/Analytics/useTracking"
-import React, { FC } from "react"
+import React, { FC, useEffect } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { resize } from "Utils/resizer"
 import { ImageLink } from "./ImageLink"
 
 interface CollectionsHubsNavProps {
   marketingHubCollections: CollectionsHubsNav_marketingHubCollections
+  collectionHubTestVariation: string
 }
 
 export const CollectionsHubsNav: FC<CollectionsHubsNavProps> = props => {
   const { trackEvent } = useTracking()
+
+  useEffect(() => {
+    const experiment = "collection_hub_entrypoints_test"
+    trackEvent({
+      action_type: Schema.ActionType.ExperimentViewed,
```

Comment:
I'm curious when and how we realized we needed to add these changes. I agree they're necessary, I just don't know when we realized it or talked about adding them.

Reply:
I knew just from implementing a/b test so much. Essentially the test isn't fully implemented without the [ExperimentViewed](https://github.com/artsy/force/tree/master/src/desktop/components/split_test#tracking-experiments) event firing.